jet-fuel/test/controllers/urls_controller_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class UrlsControllerTest < ActionController::TestCase
+  test "should create url" do
+    assert_difference('Url.count') do
+      post :create, url: {full_url: 'www.thisislong.com'}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.